docs/app/pages/getting-started/getting-started.component.html

```diff
@@ -9,6 +9,14 @@
 
     <ol>
         <li>Install <a href="https://nodejs.org/">Node.js</a>, which provides the npm package manager.</li>
+        <li>Install <a href="https://angular.io/">Angular 4</a> and <a href="https://www.typescriptlang.org/">TypeScript</a>:
+            <div>
+                <kbd>npm install @angular/common@latest @angular/compiler@latest @angular/compiler-cli@latest
```

Comment:
We should be able to get away without having to specify "@latest" on each as it should install the latest regardless? Unless there was reason for having to do this?

Also typescript is a developer dependency so should be installed using --save-dev rather than --save.